packages/scandipwa/src/component/CurrencySwitcher/CurrencySwitcher.style.scss

```diff
@@ -19,6 +19,8 @@
         padding-block-end: 1px;
     }
 
+    overflow-x: visible;
+
```

Comment:
I believe this solution isn't correct.
The currency switcher dropdown already has the `overflow` property. Here you add another one.
Instead of putting additional properties, it's better to find the existing ones and fix them.

If this is the platform issue, please use the browser stack.
I cannot reproduce it either on my Ubuntu. Maybe this is the Windows issue.

Reply:
Please remove this one as it isn't necessary.

And for the future, please do not split the properties.
We put them together before any nested block